pkg/runtimeproxy/resexecutor/cri/utils.go

```diff
@@ -22,26 +22,50 @@ import (
 	runtimeapi "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
 )
 
-func transferResource(r *runtimeapi.LinuxContainerResources) *v1alpha1.LinuxContainerResources {
-	linuxResource := &v1alpha1.LinuxContainerResources{
-		CpuPeriod:              r.GetCpuPeriod(),
-		CpuQuota:               r.GetCpuQuota(),
-		CpuShares:              r.GetCpuShares(),
-		MemoryLimitInBytes:     r.GetMemoryLimitInBytes(),
-		OomScoreAdj:            r.GetOomScoreAdj(),
-		CpusetCpus:             r.GetCpusetCpus(),
-		CpusetMems:             r.GetCpusetMems(),
-		Unified:                r.GetUnified(),
-		MemorySwapLimitInBytes: r.GetMemorySwapLimitInBytes(),
-	}
+func transferResource(r interface{}) interface{} {
```

Comment:
I really prefer to use struct explicitly instead of interface here, such transfer functions are designed for specified isolated operations, it seems we can get more info from names like transferToKoordResource and transferToCRIResource, so split this function to two isolated ones?

Reply:
Done